test/zeroD/test_zeroD.cpp

```diff
@@ -0,0 +1,57 @@
+#include <cstdio>
+#include <string>
+#include "gtest/gtest.h"
+#include "cantera/thermo.h"
+#include "cantera/kinetics.h"
+#include "cantera/zerodim.h"
+
+using namespace Cantera;
+
+//This test ensures that prior reactor initialization of a reactor does not affect later integration within a network. This example was adapted from test_reactor.py::test_equilibrium_HP.
+TEST(ZeroDim, test_individual_reactor_initialization)
+{
+    //Initial conditions
+    double T0 = 1100.0;
+    double P0 = 1013250;
+    std::string X0 = "H2:1.0, O2:0.5, AR:8.0";
+    //IdealGasConstPressureReactor solution, phase, and kinetics objects
+    std::shared_ptr<Solution> sol1 = newSolution("h2o2.yaml");
+    std::shared_ptr<ThermoPhase> gas1 = sol1->thermo();
+    std::shared_ptr<Kinetics> kin1 = sol1->kinetics();
+    gas1->setState_TPX(T0, P0, X0);
+    //Set up reactor object
+    IdealGasConstPressureReactor reactor1;
+    reactor1.setKineticsMgr(*kin1);
+    reactor1.setThermoMgr(*gas1);
+    //initialize reactor at arbitrary value
+    reactor1.initialize(0.1);
+    //Setup reactor network and integrating
+    ReactorNet network;
+    network.addReactor(reactor1);
+    network.advance(1.0);
+    //Secondary gas for comparison
+    std::shared_ptr<Solution> sol2 = newSolution("h2o2.yaml");
+    std::shared_ptr<ThermoPhase> gas2 = sol2->thermo();
+    std::shared_ptr<Kinetics> kin2 = sol2->kinetics();
+    gas2->setState_TPX(T0, P0, X0);
+    gas2->equilibrate("HP");
+    //Compare the two gases.
+    double tol = 1e-8; //relative tolerance from python assertNear test
+    EXPECT_NEAR(gas1->temperature(), gas2->temperature(), tol);
+    EXPECT_NEAR(gas1->density(), gas2->density(), tol);
+    EXPECT_NEAR(gas1->pressure(), P0, tol);
+    for(size_t i = 0; i < gas1->stateSize()-2; i++)
+    {
+        EXPECT_NEAR(gas1->massFraction(i), gas2->massFraction(i), tol);
+    }
```

Comment:
Since the test is meant to check that initializing the reactor outside a network doesn't segfault, I don't think you necessarily need the integration steps. IMO, it makes more sense to compare to things like the number of components in the reactor, etc. IMO it also makes more sense to initialize a plain `Reactor` to check the parent class directly, rather than relying on the derived classes to inherit properly and not override anything.

Reply:
I see now that @speth suggested adding it to a `ReactorNetwork` after initializing, so you should add a comment to that effect.

As one more suggestion, please limit line lengths to 88 characters (I think, see `CONTRIBUTING.md`) for the comment at the top of the test and there should be one space after the `//` for an inline comment.

Reply:
I agree. I will make some updates!

Reply:
Yep @anthony-walker, I think our comments crossed in the wires :joy:

Reply:
@anthony-walker ... thanks for updating! Could you address the one remaining comment? (see above):

> As one more suggestion, please limit line lengths to 88 characters (I think, see CONTRIBUTING.md) for the comment at the top of the test and there should be one space after the // for an inline comment.

Reply:
@bryanwweber I have limited the length in my vs-code to 88 lines, will this take care of it or is there something else you had in mind?